src/o365/spo/commands/contenttype/contenttype-field-remove.ts

```diff
@@ -0,0 +1,307 @@
+import auth from '../../SpoAuth';
+// import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+// import config from '../../../../config';
+import * as request from 'request-promise-native';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, // CommandValidate, CommandTypes,
+  CommandTypes,
+  CommandValidate,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../SpoCommand';
+import Utils from '../../../../Utils';
+import { Auth } from '../../../../Auth';
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+import config from '../../../../config';
+// import { FieldLink } from './FieldLink';
+// REST URL
+// POST http://<sitecollection>/<site>/_api/web/lists(listid)/contenttypes(contenttypeidcontenttypeid})/fieldlinks(fieldlinkid)/deleteObject()
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+
+interface Options extends GlobalOptions {
+  contentTypeId: string;
+  fieldLinkId: string;
+  webUrl: string;
+  listTitle?: string;
+  updateChildContentTypes?: boolean;
+}
+
+class SpoContentTypeFieldRemoveCommand extends SpoCommand {
+  private requestDigest: string = '';
```

Comment:
Since we need all these variables only in one command, there is no need to define them as class variables.